pandas/tests/extension/base/methods.py

```diff
@@ -138,3 +138,28 @@ def test_combine_add(self, data_repeated):
         expected = pd.Series(
             orig_data1._from_sequence([a + val for a in list(orig_data1)]))
         self.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize('frame', [True, False])
+    @pytest.mark.parametrize('periods, indices', [
+        (-2, [2, 3, 4, -1, -1]),
+        (0, [0, 1, 2, 3, 4]),
+        (2, [-1, -1, 0, 1, 2]),
+    ])
+    def test_container_shift_negative(self, data, frame, periods, indices):
```

Comment:
Very minor comment, but is there a reason this is called `test_container_shift_negative`? (because it does not only test negative periods, and 'container' seems a bit superfluous). I would simply do `test_shift`